e2e/test/scenarios/visualizations-charts/bar_chart.cy.spec.js

```diff
@@ -387,7 +403,7 @@ describe("scenarios > visualizations > bar chart", () => {
     cy.get("[data-testid^=draggable-item]").should("have.length", 0);
   });
 
-  it("should support showing data points with > 10 series (#33725)", () => {
+  it.skip("should support showing data points with > 10 series (#33725)", () => {
```

Comment:
Why do we skip this spec?

Reply:
I did not mean to leave that skipped. Thanks for catching this :heart: